pkgs/applications/window-managers/notion/default.nix

```
@@ -1,39 +1,32 @@
 {
-  enableXft ? true, libXft ? null, patches ? [], stdenv, lua, gettext, pkgconfig, xlibsWrapper, libXinerama, libXrandr, libX11,
-  xterm, xmessage, makeWrapper, fetchFromGitHub, mandoc, which
+  enableXft ? true, libXft ? null, stdenv, lua, gettext, readline, pkgconfig, xlibsWrapper, libXinerama, libXrandr, libX11,
+  xterm, xmessage, makeWrapper, fetchFromGitHub, mandoc, which, groff
 }:
 
 assert enableXft -> libXft != null;
 
 let
   pname = "notion";
-  version = "3-2017050501";
-  inherit patches;
+  version = "4.0.git";
 in
 stdenv.mkDerivation {
```

```suggestion
stdenv.mkDerivation rec {
```
pname and version can both replace `name` below, if you add `rec` here